### DISCUSSION TITLE
More ringing with SuperSlicer vs Simplify3D 4.0 on Anycubic Predator Delta 3D Printer

### DISCUSSION THREAD

**Discussion by emailcraigs** [supermerill/SuperSlicer#3750] [General]
Somebody make it make sense. The cube on the left is sliced by Superslicer and the one on the right by Simplify3D 4.0; with no input shaping on. Why does the ringing disappear so much on the S3D sliced one? As far as I know all the speed settings are very close and the acceleration settings are all set in Klipper so they should be the same. If anything the S3D is faster as it prints in 22 minutes vs the SuperSlicer printing in 23 minutes. I tried to compare the G-code files but get lost in it all. If anybody has more expertise with raw g-code I would appreciate any input. I'm attaching the two g-code files here as a comment. And since I'm being so needy, my x and y dimensional accuracy is less than .1 off but my Z is about .2 off. This is after Delta_calibrate and advanced delta calibrate. How can I fine tune this in Klipper?
![IMG_4702](https://user-images.githubusercontent.com/31557493/227385604-444abe47-f037-4caf-9288-b984c92e61ef.jpg)
![IMG_4700](https://user-images.githubusercontent.com/31557493/227385613-ffe6d096-67fc-4d63-bf73-16899b675f1f.jpg)

**Comment by linux-paul** [comment]
typically ringing just depends to acceleration/jerk that makes the frame swing a little if it is top high, Check the HW and if it is fine, klippers input shaper will usually help to achieve higher acceleration values.
